docs/frontend/index.md

```diff
@@ -0,0 +1,3 @@
+# Frontend
+
```

Comment:
The second link here is dead.

Reply:
It's dead only locally, the CI populates it with (rustdoc / odoc) contents.